plugin/src/main/js/workflow-editor.js

```diff
@@ -1,6 +1,4 @@
 import $ from 'jquery';
-// Polyfill for window.requestAnimationFrame
-import 'raf/polyfill';
```

Comment:
This was only needed before to provide `window.requestAnimationFrame` support to browsers we no longer support: https://caniuse.com/requestanimationframe